test/mocks/providers/provider_profile_page.dart

```diff
@@ -27,12 +29,22 @@ ProviderScope profileProviderScope(
     firestore: fakeFireStore,
   );
 
+  final userCommentRepo = UserCommentReposittoryService(
+    firestore: fakeFireStore,
+  );
+
+  final commentRepo = CommentRepositoryService(
+    firestore: fakeFireStore,
+  );
+
   return ProviderScope(
     overrides: [
       ...mockEmptyHomeViewModelOverride,
       firebaseAuthProvider.overrideWith(mockFirebaseAuthSignedIn),
       userRepositoryServiceProvider.overrideWithValue(userRepo),
       postRepositoryServiceProvider.overrideWithValue(postRepo),
+      userCommentRepositoryServiceProvider.overrideWithValue(userCommentRepo),
+      commentRepositoryServiceProvider.overrideWithValue(commentRepo),
```

Comment:
```suggestion
      firestoreProvider.overrideWithValue(fakeFireStore),
```

Instead of creating all the repos by hand, I suggest you just override the underlying database. If you do this you should also delete the unused variables above.

Reply:
Fixed with [4b4c5b8](https://github.com/ProximaEPFL/proxima/pull/269/commits/4b4c5b8c3fa673a557b54a4e254cfad202fa0b27)